### PR TITLE
Added git actions.

### PR DESCRIPTION
**Pull request by the-harry**

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        bundle install
+
+    - name: Run Linter (rubocop)
+      run: |
+        bundle exec rubocop --format json -o report.json
+
+    - name: Run Tests (rspec)
+      run: |
+        bundle exec rails db:create db:migrate
+        bundle exec rspec --fail-fast
```
Comment (danielmciotti) [inline .github/workflows/ruby.yml:58]: I believe we don't need the fail fast here, It's better to have the full result here and let the fail fast to development env only

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      uses: actions/checkout@v2
+
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7.1'
+
+    - name: Run Install
+      run: |
+        date
```
Comment (danielmciotti) [inline .github/workflows/ruby.yml:44]: Any reason for this?

**Review by danielmciotti — APPROVED** [review]

**Review by jairojj — APPROVED** [review]

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (the-harry) [reply .github/workflows/ruby.yml:58]: But wouldn't this consume too many CI time? 💸

File: .github/workflows/ruby.yml (hunk 2 above)
Comment (the-harry) [reply .github/workflows/ruby.yml:44]: lol heuaheauhe not that I know, I've got an example with this >< let me remove.

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (the-harry) [reply .github/workflows/ruby.yml:57]: If the cost is not a concern I can remove. What do you think?

Comment (danielmciotti) [reply .github/workflows/ruby.yml:57]: Well, I think it wouldn't be a problem 🤔 Your call here

Comment (the-harry) [reply .github/workflows/ruby.yml:57]: Idk too... @dminella you that is more familiar with our github budget, how should we proceed? 🤔

Comment (the-harry) [reply .github/workflows/ruby.yml:57]: I'll keep all suit running for now, if someone complains we change it.

**Review by DiegoGuinalia — APPROVED** [review]

**Pull request merged**